app/services/orchestration_template_dialog_service.rb

```diff
@@ -21,30 +21,10 @@ def add_stack_group(tab, position, template)
     ).tap do |dialog_group|
       add_tenant_name_field(dialog_group, 0)
       add_stack_name_field(dialog_group, 1)
-
-      if template.kind_of?(OrchestrationTemplateAzure)
-        add_azure_stack_options(dialog_group, 2)
-      elsif template.kind_of?(OrchestrationTemplateVnfd)
-        # TODO(lsmola) add Vnfd specific options
-      elsif template.kind_of?(ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplate)
-        add_availability_zone_field(dialog_group, 2)
-      else
-        add_aws_openstack_stack_options(dialog_group, 2)
-      end
+      add_deployment_options(dialog_group, 2)
```

Comment:
I'll add this method but not to raise an exception because a new manager type might not need other deployment option.